3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,86 @@
+//Age-ify (A future age calculator)
+
+const yearOfBirth = 1995;
+const yearFuture = 2045;
+console.log(typeof yearFuture);
+const curentYear = 2023;
+const age = curentYear - yearOfBirth;
+console.log(age);
+const futureAge = yearFuture - curentYear + age;
```

Comment:
Your version is also correct but you don't need to make it that complicated when you can just subtract the year of birth from the year of future :wink: 
```suggestion
const futureAge = yearFuture - yearOfBirth;
```